package.json

```diff
@@ -30,7 +30,7 @@
   "scripts" : { 
     "test": "nwbuild -r ./",
     "start": "nwbuild -r ./",
-    "preinstall": "npm list nw-builder -g || npm install nw-builder -g",
+    "prebuild": "npm list nw-builder -g || npm install nw-builder -g",
```

Comment:
could you change this back to preinstall?

as this requires nw-builder to be installed to use `npm test` or `npm start`.

or we could do 
```json
"pretest": "npm list nw-builder -g || npm install nw-builder -g",
"prestart": "npm list nw-builder -g || npm install nw-builder -g",
"prebuild": "npm list nw-builder -g || npm install nw-builder -g"
```

rather than 
```json
"preinstall": "npm list nw-builder -g || npm install nw-builder -g"
```